fontforgeexe/pythonui.c

```diff
@@ -596,12 +597,12 @@ static PyObject *PyFFFont_addGtkWindowToMainEventLoop(PyFF_Font *self, PyObject
     if ( !PyArg_ParseTuple( args, "i", &v ))
         return( NULL );
 
-    gpointer gdkwindow = gdk_xid_table_lookup( v );
```

Comment:
Oh -- these changes are unrelated to the functionality. They're basically just to get this part of the code to compile. It would take more work to revive the gxdraw collab hooks.